### DISCUSSION TITLE
这个地址https://d.injdk.cn/是基于本项目Alist的二开吗

### DISCUSSION THREAD

**Discussion by jzyyws** [AlistGo/alist#7280] [General]
今天在网络上下载JDK，发现一个地址https://injdk.cn/，下载地址提供的文件列表服务https://d.injdk.cn/ 是基于本项目Alist的二开吗？

**Comment by jzyyws** [comment]
https://injdk.cn/
https://d.injdk.cn/

**Comment by anwen-anyi** [comment]
这是AList V2版本

后端：https://github.com/alist-org/v2  （应该是这个？）
前端：https://github.com/alist-org/web-v2

**Comment by jiqing112** [comment]
明确回复你，是的
